wurst/systems/core/init/Units.wurst

```diff
@@ -0,0 +1,33 @@
+package Units
+import ClosureForGroups
+
+public destructable MAMMOTH_GATE
+public unit MAMMOTH = null
+public unit MASTER_CRAFTER = null
+public unit OMINOUS_ALTAR = null
```

Comment:
Looks like we only have one reference to `OMINOUS_ALTAR`--I wonder if we should move this initialization to that location

https://github.com/island-troll-tribes/island-troll-tribes/blob/master/wurst/systems/core/Game.wurst#L38

Reply:
Which we should probably move into https://github.com/island-troll-tribes/island-troll-tribes/blob/master/jass/crafting/Hydra%20Altar%20Buyingmake.j once we move it to Wurst